tutor/src/models/subject-order.ts

```diff
@@ -0,0 +1,8 @@
+export const SubjectOrder = [
+    'Math',
+    'Science',
+    'Social Sciences',
+    'Humanities',
+    'Business',
+    'AP® Courses',
+]
```

Comment:
I wonder if there is a way to persist the order from the database, instead of in-memory.

Reply:
Yeah that'd be nice, maybe something we can add to admin sometime (might be nice to create a Subject model then too.)